eng/Version.Details.xml

```diff
@@ -9,305 +9,305 @@
-      <Sha>72b7d236ad634c2280c73499ebfc2b594995ec06</Sha>
+      <Sha>162b33e19eb5a3434867541be691e1082039b130</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.EventLog" Version="6.0.0-alpha.1.20560.10">
+    <Dependency Name="Microsoft.Extensions.Logging.EventLog" Version="6.0.0-alpha.1.20565.4">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>72b7d236ad634c2280c73499ebfc2b594995ec06</Sha>
+      <Sha>162b33e19eb5a3434867541be691e1082039b130</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging.TraceSource" Version="6.0.0-alpha.1.20560.10">
+    <Dependency Name="Microsoft.Extensions.Logging.TraceSource" Version="6.0.0-alpha.1.20565.4">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>72b7d236ad634c2280c73499ebfc2b594995ec06</Sha>
+      <Sha>162b33e19eb5a3434867541be691e1082039b130</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging" Version="6.0.0-alpha.1.20560.10">
+    <Dependency Name="Microsoft.Extensions.Logging" Version="6.0.0-alpha.1.20565.4">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>72b7d236ad634c2280c73499ebfc2b594995ec06</Sha>
+      <Sha>162b33e19eb5a3434867541be691e1082039b130</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0-alpha.1.20560.10">
+    <Dependency Name="Microsoft.Extensions.Options.ConfigurationExtensions" Version="6.0.0-alpha.1.20565.4">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>72b7d236ad634c2280c73499ebfc2b594995ec06</Sha>
+      <Sha>162b33e19eb5a3434867541be691e1082039b130</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options.DataAnnotations" Version="6.0.0-alpha.1.20560.10">
+    <Dependency Name="Microsoft.Extensions.Options.DataAnnotations" Version="6.0.0-alpha.1.20565.4">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>72b7d236ad634c2280c73499ebfc2b594995ec06</Sha>
+      <Sha>162b33e19eb5a3434867541be691e1082039b130</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Options" Version="6.0.0-alpha.1.20560.10">
+    <Dependency Name="Microsoft.Extensions.Options" Version="6.0.0-alpha.1.20565.4">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>72b7d236ad634c2280c73499ebfc2b594995ec06</Sha>
+      <Sha>162b33e19eb5a3434867541be691e1082039b130</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Primitives" Version="6.0.0-alpha.1.20560.10">
+    <Dependency Name="Microsoft.Extensions.Primitives" Version="6.0.0-alpha.1.20565.4">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>72b7d236ad634c2280c73499ebfc2b594995ec06</Sha>
+      <Sha>162b33e19eb5a3434867541be691e1082039b130</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Internal.Transport" Version="6.0.0-alpha.1.20560.10">
+    <Dependency Name="Microsoft.Extensions.Internal.Transport" Version="6.0.0-alpha.1.20565.4">
```

Comment:
Getting consistent errors using this version in global.json. Has dotnet/runtime stopped producing archives usable with `dotnet-install.*` scripts @mmitche❔ e.g.
``` text
dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Runtime/6.0.0-alpha.1.20565.4/dotnet-runtime-6.0.0-alpha.1.20565.4-linux-x64.tar.gz
curl: (22) The requested URL returned error: 404 Not Found
```